tests/test_scan.py

```diff
@@ -11,7 +12,10 @@
 def test_cd_context_manager():
     prev = getcwd()
     with cd("/tmp"):  # nosec
-        assert getcwd() == "/tmp"  # nosec
+        if platform.system() == "Darwin":
+            assert getcwd() == "/private/tmp"  # nosec
+        else:
+            assert getcwd() == "/tmp"  # nosec
```

Comment:
I think this can be reworked to avoid platform-specific code like this (untested):
```python
target = os.path.realpath("/tmp")
with cd(target):
    assert getcwd() == target
assert getcwd() == prev
```

Reply:
Agree with Agateau-gg. Otherwise it looks good to me :+1: